apps/admin-ui/components/Layout/Header/Breadcrumbs.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Breadcrumbs, Link, Typography } from "@mui/material";
+import { BreadcrumbsNameMaps } from "./Header";
+import { useRouter } from "next/router";
+
+interface BreadcrumbProps {
+    breadcrumbs: BreadcrumbsNameMaps
+}
+
+export default function Breadcrumb({ breadcrumbs: {
+    title,
+    links
+} }: BreadcrumbProps) {
+    const { pathname } = useRouter()
+    const pathnames = pathname.split('/').filter((x) => x)
```

Comment:
what is the essence of the filter at the end?
from what i see, it only takes a value and return it, is that not a redundant operation?

also, you can't call this variable ~pathnames~, you can't split `a` pathname and have multiple pathnames.
it's the `subPaths` or simply `paths`.

Reply:
The filter JS function here is used to remove the empty string which appeared when split.
Actually, I expected the same from the filter function, like taking a value and returning it, so it wasn't useful, but when split, a string appeared empty in the array. So we've added a filter to perform an array without an empty string value.